package.json

```diff
@@ -50,7 +50,19 @@
     "parserOptions": {
       "parser": "@babel/eslint-parser"
     },
-    "rules": {}
+    "rules": {
+      "vue/multi-word-component-names": [
+        "error",
+        {
+          "ignores": [
```

Comment:
Please remove as we dont have Home component
  ```
        "ignores": [
            "Home"
          ]

```

Reply:
Done.